raiden_contracts/tests/fixtures/channel.py

```diff
@@ -359,7 +358,9 @@ def get(
             participant2_values,
         )
 
-        mine_blocks(web3, settle_timeout)
+        web3.provider.ethereum_tester.time_travel(  # type: ignore
+            web3.eth.get_block("latest").timestamp + settle_timeout + 1  # type: ignore
+        )
```

Comment:
I know this is my own code. I probably should just do it myself. But do you agree that it makes sense to shim this down to a utility function? Would also remove the need to always put this stupid `# type: ignore` and maybe save some imports of the `TEST_SETTLE_TIMEOUT` variable. Definitely should improve readability.

Reply:
Duplicate of [this comment](https://github.com/raiden-network/raiden-contracts/pull/1573/files?authenticity_token=Kar%2BtamiGrDSqMZE9CYL98ryCissNBtNCOrxdBhZaAbFmXQ%2BYMIOwCGEXVJQ%2FU39lqwKk4i4r2nzR%2FXd0eyN5A%3D%3D&file-filters%5B%5D=.json&file-filters%5B%5D=.py&hide-deleted-files=true#r769468919). Please just close it when done. Just keep it for the content. 🤣